x-pack/test/accessibility/config.ts

```diff
@@ -15,14 +15,15 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     ...functionalConfig.getAll(),
 
     testFiles: [
-      require.resolve('./apps/login_page'),
-      require.resolve('./apps/home'),
-      require.resolve('./apps/grok_debugger'),
-      require.resolve('./apps/search_profiler'),
-      require.resolve('./apps/uptime'),
-      require.resolve('./apps/spaces'),
-      require.resolve('./apps/advanced_settings'),
-      require.resolve('./apps/dashboard_edit_panel'),
+      // require.resolve('./apps/login_page'),
+      // require.resolve('./apps/home'),
+      // require.resolve('./apps/grok_debugger'),
+      // require.resolve('./apps/search_profiler'),
+      // require.resolve('./apps/uptime'),
+      // require.resolve('./apps/spaces'),
+      // require.resolve('./apps/advanced_settings'),
+      // require.resolve('./apps/dashboard_edit_panel'),
+      require.resolve('./apps/users'),
     ],
```

Comment:
Remember to uncomment these before merging!

Reply:
// https://github.com/elastic/eui/issues/2841
    it.skip('a11y test for Change password screen', async () => {